project_event/views/res_partner_category_view.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="action_partner_category" model="ir.actions.act_window">
+        <field name="name">Client Type</field>
+        <field name="res_model">res.partner.category</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+
+    <record id="res_partner_category_form" model="ir.ui.view">
+        <field name="name">Category Form</field>
+        <field name="model">res.partner.category</field>
+        <field name="inherit_id" ref="base.view_partner_category_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='parent_id']" position="after">
+                <field name="client_type"/>
+            </xpath>
+        </field>
+    </record>
+
+    <menuitem id="menu_res_partner_category"
```

Comment:
not the correct action, put this menu in the res_partner_category_type view with the correct action

Reply:
@rimbendhaou  Done